package.json

```diff
@@ -75,6 +75,9 @@
     "testdouble": "3.16.6",
     "xo": "0.32.1"
   },
+  "overrides": {
+    "semantic-release": "20.0.0-beta.1"
+  },
```

Comment:
i think we should target removing this before promoting the pre-release of core to stable

Reply:
TIL! I didn't know `overrides` was a thing! It's only supported in npm 8.3+ though, right?
https://www.stefanjudis.com/today-i-learned/how-to-override-your-dependencys-dependencies/

Or was it backported?